backend/build.gradle

```diff
@@ -89,6 +89,7 @@ subprojects {
 
     jar {
         reproducibleFileOrder = true
+        zip64 = true
```

Comment:
remove it, but in the shadowJar this is needed, because the number of files were to large for a regular zip. Therefor it failed when building on the preview server